lib/iris/coords.py

```diff
@@ -1649,8 +1648,8 @@ def bounds(self):
         """
         if self._bounds is not None:
             bounds = self._bounds
-            if isinstance(bounds, biggus.Array):
-                bounds = bounds.ndarray()
+            if is_lazy_data(bounds):
+                bounds = bounds.compute()
```

Comment:
Perhaps @pp-mo `as_concrete` follow-on PR might make this melt away? @pp-mo is that the case?